### DISCUSSION TITLE
Finn and Fin-Experimental Installing Error

### DISCUSSION THREAD

**Discussion by Ba1tu3han** [Xilinx/finn#754] [Q&A]
I'm following the getting started page and I have done all steps. I'm using the pre-built image. When I run "1-train-mlp-with-brevitas" I get an error which is shown in picture below. 

Additionally, in terminal I realized that finn and finn-experimental are not installed properly. 

My Vivado version is 2022.1, Finn V0.8.1 (cloned), Ubuntu 20.4.

I could not fix it. Could you help me? What would you recommend me?

Thanks,
Batuhan,

![Screenshot from 2023-02-06 22-07-19](https://user-images.githubusercontent.com/29502318/217062450-c4bd5fe7-2d12-4f7a-87c4-987fd98da27f.png)

![Screenshot from 2023-02-06 22-00-28](https://user-images.githubusercontent.com/29502318/217062472-1ed2c36d-335c-40a4-a920-c669a1b890b5.png)

![Screenshot from 2023-02-06 22-00-14](https://user-images.githubusercontent.com/29502318/217062485-04fca6af-0011-45b4-a233-50515dc00bc4.png)

**Comment by Xuexiabc** [comment]
I met the same issue with you, it was solved by using "./run-docker.sh notebook", rather than "sudo ./run-docker.sh notebook".

**Reply by Ba1tu3han** [reply]
Hello,

I have tried it using "./run-docker.sh notebook" but it is still giving the same error. Do you have any other recommendation? 
![image](https://user-images.githubusercontent.com/29502318/221408942-07c36195-4c88-4fee-b622-644f4afa07c1.png)

**Comment by Ba1tu3han** [comment]
Update:

The problem is not solved.

I have tried prebuilt image and it has the same problem. 

I use following commands.

sudo docker pull maltanar/finn:dev_latest
FINN_DOCKER_TAG=maltanar/finn:dev_latest
FINN_DOCKER_PREBUILT=1

How can I solve it?

![image](https://user-images.githubusercontent.com/29502318/221413624-9cf94111-c863-4967-b1b5-b60ab41452fa.png)

**Reply by Ba1tu3han** [reply]
@auphelia Could you set this discussion as open? Thanks.
